enclave_manager/Makefile

```diff
@@ -36,12 +45,14 @@ build :
 
 install:
 	@echo INSTALLING WHEEL FILE =================
-	pip3 install $(WHEEL_FILE)
+	pip3 install $(ENCLAVE_MANAGER_WHEEL_FILE) $(ENCLAVE_COMMON_WHEEL_FILE)
 
 clean:
-	if pip3 uninstall --yes $(WHEEL_FILE); then echo UNINSTALLED $(WHEEL_FILE) WHEEL FILE ; fi
+	if pip3 uninstall --yes $(ENCLAVE_MANAGER_WHEEL_FILE) $(ENCLAVE_COMMON_WHEEL_FILE); then echo UNINSTALLED $(ENCLAVE_MANAGER_WHEEL_FILE) $(ENCLAVE_COMMON_WHEEL_FILE) WHEEL FILE ; fi
```

Comment:
I suggest breaking this long line up for readability. Such as  the following (begin each line with a tab character):

```
	if pip3 uninstall --yes $(ENCLAVE_MANAGER_WHEEL_FILE) $(ENCLAVE_COMMON_WHEEL_FILE); then \
	    echo UNINSTALLED $(ENCLAVE_MANAGER_WHEEL_FILE) $(ENCLAVE_COMMON_WHEEL_FILE) WHEEL FILE ; fi
```